pyflakes/test/test_api.py

```diff
@@ -447,19 +447,26 @@ def evaluate(source):
             self.fail()
 
         with self.makeTempFile(source) as sourcePath:
+            message = 'invalid syntax'
+            column = 11
+            whitespace_before_caret = 10
             if PYPY:
                 message = 'end of file (EOF) while scanning triple-quoted string literal'
-            else:
-                message = 'invalid syntax'
+            elif sys.version_info >= (3, 9):
+                message = 'invalid string prefix'
+                column = 52
+                whitespace_before_caret = 14
```

Comment:
these offsets and error message seem like a bug, could you report this to upstream?  there is no line 14 and column 52 for starters and it's unclear why it's talking about a string prefix when the error is an unterminated string literal

Reply:
The reason it is talking about a string prefix is that that's the first error in encounters. The invalid string prefix is "quux".

Reply:
I don't know what to report to upstream. When I run the code, Python 3.9.0a5 gives me
```
  File "/mnt/c/Users/Admin/Desktop/syntaxerror.py", line 6
    '''quux'''
          ^
SyntaxError: invalid string prefix
```
which is correct.

And if I catch as print the exc_info I get
```
(<class 'SyntaxError'>, SyntaxError('invalid string prefix', ('<string>', 7, 50, "    '''\ndef bar():\n    pass\ndef baz():\n    '''quux'''\n")), <traceback object at 0x7fb1c824b340>)
```
This is identical to the system Python 3.6

I have no idea what the tests in this file are for, so can't have an opinion about what is reasonable here.

Reply:
that's fine, I've gone ahead and made the report myself :) https://bugs.python.org/issue40335

Reply:
Aha! That's why I got the same with Python 3.6. Should have tried 3.8 as well